src/Runner.jl

```diff
@@ -631,6 +631,14 @@ function generate_compiler_wrappers!(platform::AbstractPlatform; bin_path::Abstr
     end
     gofmt(io::IO, p::AbstractPlatform) = wrapper(io, "/opt/$(host_target)/go/bin/gofmt"; allow_ccache=false)
 
+    # OCaml stuff
+    function ocaml_wrapper(io::IO, tool::String, p::AbstractPlatform)
+        return wrapper(io, "/opt/$(aatriplet(p))/bin/$(aatriplet(p))-$(tool)")
```

Comment:
This path is wrong, but I wasn't quite sure where the RootFS artifacts would end up